.github/workflows/build.yml

```diff
@@ -0,0 +1,14 @@
+name: Build
+on:
```

Comment:
This action is unnecessary, we already have the Makefile CI action.

https://github.com/vinegarhq/vinegar/actions/workflows/makefile.yml

Reply:
............. it replaces it.

i hadn't did `git add` properly.

Reply:
it did replace it dude lol